dawn/webpack.production.config.js

```diff
@@ -1,62 +0,0 @@
-const path = require('path');
-const webpack = require('webpack');
-
-const modules = {
-  preLoaders: [
-    { test: /\.js$/, loader: 'eslint-loader', exclude: /node_modules/ }
-  ],
-  loaders: [
-    {
-      test: /\.js$/,
-      exclude: /node_modules/,
-      loader: "babel-loader",
-      query: {
-        presets: [ 'es2015', 'react' ],
-        plugins: [ 'transform-object-rest-spread', 'transform-regenerator' ]
-      }
-    },
-    {
-      test: /\.json$/,
-      loader: 'json-loader'
-    }
-  ]
-};
-
-const plugins = [
-  new webpack.DefinePlugin({
-    VERSION: JSON.stringify(require('./package.json').version)
-  }),
-  new webpack.DefinePlugin({
-    "global.GENTLY": false
-  }),
-  new webpack.DefinePlugin({
-    'process.env': {
-      'NODE_ENV': '"production"'
```

Comment:
We keep a production variable to differentiate what is being developed vs what should be shipped to students